packages/neo-one-node-protocol/src/consensus/ConsensusContext.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+import { utils } from '@neo-one/utils';
+
+export default class ConsensusContext {
+  fastForwardSeconds: number;
+
+  constructor({ fastForwardSeconds }: {| fastForwardSeconds: number |}) {
+    this.fastForwardSeconds = fastForwardSeconds;
+  }
+
+  currentCustomTime(): number {
```

Comment:
Call this `nowSeconds`, as far as consensus is concerned we're giving it the current time.